data/meson.build

```diff
@@ -0,0 +1,11 @@
+# install_data(
+#     'com.github.needleandthread.vocal.gschema.xml',
+#     install_dir: get_option('datadir') / 'metainfo',
+#     rename: meson.project_name() + '.gschema.xml',
+# )
+
+install_data(
+    'com.github.needleandthread.vocal.gschema.xml',
+    install_dir: get_option('datadir') / 'glib-2.0' / 'schemas',
+    rename: meson.project_name() + '.gschema.xml',
```

Comment:
What is the purpose of the rename option here? It's the same name either way, surely?